backend/models/students.js

```diff
@@ -0,0 +1,35 @@
+import { Model } from 'sequelize';
+
+export default (sequelize, DataTypes) => {
+  class Students extends Model {}
+  Students.init(
```

Comment:
Sorry I didn't catch this at first but can you make the all instances (including the file name) of this model singular in this file?